carmin.yaml

```diff
@@ -298,7 +298,27 @@ paths:
               schema:
                 $ref: '#/components/schemas/Pipeline'
         default: 
-          $ref: '#/components/responses/Error'      
+          $ref: '#/components/responses/Error'
+  /pipelines/{pipelineIdentifier}/boutiquesdescriptor:
+    get:
+      summary: Returns the Boutiques descriptor of the pipeline, if available. 
+      operationId: getBoutiquesDescriptor
+      parameters:
+      - name: pipelineIdentifier
+        in: path
+        required: true
+        schema:
+          type: string
+          format: ascii
+      responses:
+        '200':
+          description: successful response
+          content:
+            application/json:
+              schema:
+                $ref: 'https://raw.githubusercontent.com/boutiques/boutiques/master/tools/python/boutiques/schema/descriptor.schema.json'
```

Comment:
We should at least put a link on a fixed version of boutique.
Have you tried to see if a swagger tool (or swaggerhub) manage to deal with an external link ?

Reply:
The master branch of boutiques contains the latest stable version of the spec (development is in 'develop').  I thought that this would be better than a fixed version because that would avoid having to update carmin for every new version of boutiques. The boutiques versions are most of the time backward compatible. But if you really want a fixed version I'm fine.

Reply:
It would bother me if the CARMIN specification contains something vague. 2 CARMIN servers with the same version could  have 2 different behaviors of the getBoutiquesDescriptor methode because one updated to the last boutiques version and not the other. A CARMIN client communicating with the 2 servers could be lost, event if it's backward compatible. And if it's not backward compatible (that could happen one day, even if you don't want to at the moment), then it's not only confusing but harmful.

I could accept to have an open "latest" link if the supported boutiques version is advertised in the platform properties. Then a client could say : "This server has a new version of boutiques, it could break something".

A fixed version has the drawback to force us to update the CARMIN version if we want to support a new boutiques version. That's fine to me.